doc/source/conf.py

```diff
@@ -1,44 +1,34 @@
 # -*- coding: utf-8 -*-
```

Comment:
The ITWM and BDSS conf.py are exactly the same, except for the plugin version import thingy (one line). The nevergrad plugin was the same, except the order of the lines was different in places. Therefore I just copy and pasted the ITWM, only changing the version line.

Reply:
The reason why I changed it was because the docs was complaining it couldn't find the enthought-sphinx thingy (template? style?).

Reply:
Yes, if you check lines 69-82 you can see what's happening - a warning will be raised if there's no Enthought sphinx theme (which, since we are not using it for these docs we can remove it). 

There are some additional minor changes other than the version, (`MOCK_MODULES`, `project`, `copyright`) but I don't think for this case it makes much difference. I would recommend keeping the `'traits.util.trait_documenter'` extension though, since we really should be using this in the other repos anyway.

Reply:
Interestingly, the sphinx-theme was only a problem for the linux build, not on mac.